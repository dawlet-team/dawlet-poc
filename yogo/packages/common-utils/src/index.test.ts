import { fetchAvailableDawlets, readPackagesFolder, filterUnrelatedFolders } from './'
describe('fetchAvailableDawlets', () => {
  it('reads packages folder', () => {
    const dawlets = readPackagesFolder()
    expect(dawlets).toMatchSnapshot()
  })
  it('filters out unrelated folders', () => {
    const folders = ['algolet', 'midilet', 'common-utils', 'core', 'launcher', 'react-electron-webpack-config']
    const filtered = filterUnrelatedFolders(folders)
    expect(filtered).toEqual(['algolet', 'midilet'])
  })
  it('fetches available dawlets', () => {
    expect(fetchAvailableDawlets()).toMatchSnapshot()
  })
})