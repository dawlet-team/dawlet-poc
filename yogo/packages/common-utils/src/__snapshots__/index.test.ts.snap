// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fetchAvailableDawlets fetches available dawlets 1`] = `
Array [
  "algolet",
]
`;

exports[`fetchAvailableDawlets reads packages folder 1`] = `
Array [
  "algolet",
  "common-utils",
  "core",
  "launcher",
  "react-electron-webpack-config",
]
`;
